etc/calc_release_version.py

```diff
@@ -152,77 +180,104 @@ def get_next_minor(prerelease_marker):
                               str(version_new['patch']) + '-' + \
                               version_new['prerelease']
             new_version_parsed = parse_version(new_version_str)
-            if new_version_parsed > version_parsed:
+            if new_version_parsed > version_parsed: # type: ignore
                 version_str = new_version_str
                 version_parsed = new_version_parsed
-                if DEBUG:
-                    print('Found new best version "' + version_str \
+                debug('Found new best version "' + version_str \
                             + '" based on branch "' \
                             + release_branch_match.group('brname') + '"')
     return version_str
 
-def get_branch_tags(active_branch_name):
+def get_branch_tags(active_branch_name):  # type: (str) -> list[str]
     """
-    Returns the tag or tags (as a single string with newlines between tags)
-    corresponding to the current branch, which must not be master.  If the
-    specified branch is a release branch then return all tags based on the
-    major/minor X.Y release version.  If the specified branch is neither master
-    nor a release branch, then walk backwards in history until the first tag
-    matching the glob 'r*' and return that tag.
+    Returns a list of tags corresponding to the current branch, which must not
+    be master.  If the specified branch is a release branch then return all tags
+    based on the major/minor X.Y release version.  If the specified branch is
+    neither master nor a release branch, then walk backwards in history until
+    the first tag matching the glob 'r*' and return that tag.
     """
 
+    debug('Getting tags for branch {0}'.format(active_branch_name))
     if active_branch_name == 'master':
         raise Exception('this method is not meant to be called while on "master"')
-    tags = ''
+
     release_branch_match = RELEASE_BRANCH_RE.match(active_branch_name)
     if release_branch_match:
         # This is a release branch, so look for tags only on this branch
-        tag_glob = 'r' + release_branch_match.group('vermaj') + '.' \
+        tag_glob = 'r'+  release_branch_match.group('vermaj') + '.' \
```

Comment:
```suggestion
        tag_glob = 'r' + release_branch_match.group('vermaj') + '.' \
```
I'm pretty sure that this was an inadvertent change.